common/Inc/stm32xx_hal.h

```diff
@@ -24,4 +24,9 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+// Init function prototypes
+void Error_Handler(void);
+void HAL_MspInit(void);
+void SystemClock_Config(void);
+
```

Comment:
MspInit and ClockConfig are already in `stm32x4xx_hal_init.h`. Lowk, I think it makes more sense for them to be here so we can remove the former.